netcode.c

```diff
@@ -4172,6 +4210,10 @@ int skillz_add_client_to_match(struct netcode_server_t * server, uint64_t skillz
         match->num_clients_in_match++;
```

Comment:
Re: `if ( match->num_clients_in_match >= server->max_clients_per_match )` above.  Could we get into a state where a client tries to re-connect before disconnecting gracefully?

Reply:
No, sure netcode will not allow for that behavior by default.  'Re-connection' is just sending a connect request.  If they are already connected then their id is stored and netcode does not allow them to continue with the connection.